builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +125,25 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
+	vmxpath := filepath.Join(d.AppPath, "Contents", "Library", "vmware-vmx")
+	var stderr bytes.Buffer
+	cmd := exec.Command(vmxpath, "-v")
+	cmd.Stderr = &stderr
+	versionRe := regexp.MustCompile(`(?i)VMware [a-z0-9-]+ (\d+)\.`)
+	matches := versionRe.FindStringSubmatch(stderr.String())
+	if matches[1] >= "13" {
+		if runtime.GOOS == "windows" && runtime.GOARCH == "arm64" {
```

Comment:
If GOOS == "windows" is this correct? I thought Fusion only ran on MacOs. 

Is this check supposed to be related to the windows guest os?

Reply:
That is correct I mistakenly was thinking runtime.GOOS but it actually should be the guest OS. I'm not sure how to test for guest OS though. My googling found the runtime.GOOS which is what I went with not thinking that would be the Host and not Guest.